backend/Dockerfile

```diff
@@ -5,7 +5,10 @@ WORKDIR /app
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
-COPY  . /app/backend/.
```

Comment:
Is it possible to add the clamav parts to the requirements. txt file? If so, we should do that.